tests/test_data_labeled.xml

```diff
@@ -65,4 +65,14 @@
   <Name><CorporationName>ENGLEWOOD</CorporationName> <CorporationName>MARKET,</CorporationName> <CorporationLegalType>INC.</CorporationLegalType></Name>
   <Name><CorporationName>GRABTAXI</CorporationName> <CorporationName>HOLDINGS,</CorporationName> <CorporationLegalType>PTE</CorporationLegalType> <CorporationLegalType>LTD</CorporationLegalType></Name>
   <Name><CorporationName>VIKI</CorporationName> <CorporationName>ENGINEERING</CorporationName> <CorporationLegalType>PTE.</CorporationLegalType> <CorporationLegalType>LTD.</CorporationLegalType></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>LARA</GivenName> <MiddleInitial>E</MiddleInitial> <Surname>STROTHEIDE</Surname></Name>
```

Comment:
In general we try not to use the same samples in the training and testing sets. Do you mind removing the corresponding samples from the training data for these tests?

Reply:
Yes, I will

Reply:
@az0, thanks for picking this back up again. Did you address this?

Reply:
@fgregg Yes, I removed STROTHEIDE from the training set in this commit https://github.com/az0/probablepeople/commit/4e88092462e296f51cbfdfee6dc0256836b373ad